packages/connect-query-core/src/create-infinite-query-options.ts

```diff
@@ -185,6 +185,7 @@ export function createInfiniteQueryOptions<
     schema,
     transport,
     input,
+    pageParamKey,
```

Comment:
Technically, the only change that is actually required. Everything else is optional enhancement to make sure this doesn't happen again